packages/components/src/components/table-stateful/shadow.tsx

```diff
@@ -512,19 +512,26 @@ export class KolTableStateful implements TableAPI {
 		}
 	}
 
-	private getSelectedData(selectedKeys: string[]): KoliBriTableDataType[] {
-		if (this.state._selection) {
-			const keyPropertyName = this.state._selection.keyPropertyName ?? 'id';
-			if (keyPropertyName) return this.state._sortedData.filter((item) => selectedKeys.includes(item[keyPropertyName] as string));
+	private getSelectedData(selectedKeys: string[] | string): null | KoliBriTableDataType | KoliBriTableDataType[] {
```

Comment:
Sollte `selectedKeys` (plural) nicht immer ein Array sein, wozu brauchen wir hier den `string` type? In der Verwendung (Sample App) hatte ich gesehen, dass die Selektion immer eine Array ist, auch bei `multiple: false`. Das fand ich okay und würde es so durchziehen.

Reply:
Ja, am besten mal in der größeren Runde, z.B. im Daily. Ich fände wichtig, dass Event und `getSelection` einheitlich sind. Mit Präferenz auf `Item` statt `Item[]` im single-selection-Modus.